### PR TITLE
ci: Improve physmon report comment

### PR DESCRIPTION
**Pull request by paulgessinger**
With these changes, if the physmon job didn't complete successfully, the report comment will look something like this:

image

### CODE REVIEW

File: CI/physmon/generate_comment.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+expected_reports = {""}
```
Comment (benjaminhuth) [inline CI/physmon/generate_comment.py:3]: Maybe add a comment whats the reasoning behind this? I think that would improve future maintainability...

**Review by benjaminhuth — APPROVED** [review]
Very nice! just a few very minor comments, feel free to ignore them

File: CI/physmon/generate_comment.py (hunk 1 above)
Comment (paulgessinger) [reply CI/physmon/generate_comment.py:3]: Ah this needs to go.

File: CI/physmon/generate_comment.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+_, artifact_dir, outfile = sys.argv
+artifact_dir = Path(artifact_dir)
+outfile = Path(outfile)
+
+artifact_url = os.environ["ARTIFACT_URL"]
+pr_sha = os.environ["PR_SHA"]
+
+
+print("Hi from python")
```
[the remark this answers is not in the document]
Comment (paulgessinger) [reply CI/physmon/generate_comment.py:40]: This needs to go too.

[the remark this answers is not in the document]
Comment (paulgessinger) [reply CI/physmon/generate_comment.py:40]:
```suggestion
```

File: CI/physmon/generate_comment.py (hunk 1 above)
Comment (paulgessinger) [reply CI/physmon/generate_comment.py:3]:
```suggestion
```

**Review by AJPfleger — APPROVED** [review]

**Pull request merged**